cmd/git-sync/main.go

```diff
@@ -184,7 +275,7 @@ func envInt(key string, def int) int {
 	if env := os.Getenv(key); env != "" {
 		val, err := strconv.ParseInt(env, 0, 0)
 		if err != nil {
-			log.Error(err, "invalid env value, using default", "key", key, "val", os.Getenv(key), "default", def)
+			fmt.Fprintf(os.Stderr, "invalid env value: %v, using default, key=%q, val=%q, default=%d\n", err, key, os.Getenv(key), def)
```

Comment:
This is a good catch - it would log before flag parsing (bad).

How about:

```
fmt.Fprintf(os.Stderr, "WARNING: invalid env value (%v): using default, key=%s, val=%q, default=%d\n", err, key, env, def)
```